### ISSUE TITLE
LSTMBlockCell variable names changed - old checkpoints cannot be restored

### ISSUE THREAD

**Issue by albertz** [tensorflow/tensorflow#11168]
In TF 1.1 and earlier, bias and weights had the variable names `"biases"` and `"weights"` in the `LSTMBlockCell`.

Since TF 1.2, they have the variable names `"bias"` and `"kernel"`.

Why was that changed? It makes all checkpoint files incompatible, but for no real reason.

**Comment by ali01** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by albertz** [comment]
@ali01  No, you misunderstood. This is a bug in TensorFlow. I did not asked any question. The "why" is a rhetorical question. Of course it should not have been changed. There is no reason to just change the variable names to make checkpoint files incompatible.

**Comment by albertz** [comment]
Note: I checked the commit logs, and there have been several breaking changes. First for TF 1.0 it was changed to what I was expecting in commit 92da8abfd35b93488ed7a55308b8f589ee23b622. Then the breaking change I was talking about here was introduced in 157370e5916b85c65958ed8383ae31d727228ed7. My argument in this GitHub issue was that this should not have been done. There is really no point in posting this to StackOverflow.
